### ISSUE TITLE
Термин может присутствовать в модели в нижнем регистре, а в датасете - в верхнем

### ISSUE THREAD

**Issue by aatimofeev** [issekifora/ml_w2v_evaluate#1]
И поэтому не найдётся

**Comment by alexandraizhevskaya** [comment]
Да, тестировала на модели, где все было в нижнем регистре, и не обратила на это внимание. Так как данные могут быть разные, можно просто вставить в try дополнительные блоки с upper() и capitalize(), чтобы проверять тогда все возможные случаи
